### PR TITLE
Community resize server

### PR DESCRIPTION
**Pull request by stephamon**
JIRA 3151

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by stephamon** [comment]
@cyrichardson can I have an edit, please?

**Comment by cyrichardson** [comment]
@stephamon On it!

**Comment by cyrichardson** [comment]
@stephamon Some suggestions here: 
[resize-up-compute-and-memory-servers_cyrEdits2.docx](https://github.com/rackerlabs/rackspace-how-to/files/2832780/resize-up-compute-and-memory-servers_cyrEdits2.docx)

**Comment by cyrichardson** [comment]
@stephamon I think the link for Reboot your server might need a link update - https://support.rackspace.com/how-to/reboot-a-server/

**Comment by cyrichardson** [comment]
@stephamon Same with link about resetting password - https://support.rackspace.com/how-to/reset-a-server-password/.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d565344c1d/how-to/)

File: content/cloud-servers/resize-up-compute-and-memory-servers.md [hunk 1]
```diff
@@ -70,16 +72,51 @@ Use the following steps to resize up your servers:
...
 
 ### Additional resources
 
 For more information, see the following resources:
 
 - [Cloud Servers API reference: Resize specified server](https://developer.rackspace.com/docs/cloud-servers/v2/api-reference/svr-basic-operations/#resize-specified-server)
-- [How to Resize Compute and Memory Flavor Servers: Pitchfork method](https://community.rackspace.com/general/f/general-discussion-forum/8567/how-to-resize-compute-and-memory-flavor-servers) (scroll to "Pitchfork method" section)
-- [Reset your server password](/how-to/reset-your-server-password)
+- [Pitchfork - the Rackspace Cloud API web application](/how-to/pitchfork-the-rackspace-cloud-api-web-application)
+- [Reset your server password](/how-to/reset-a-server-password)
```
Comment (RackerKate) [inline content/cloud-servers/resize-up-compute-and-memory-servers.md]: Suggest matching link text to destination page name. It changed a bit.

File: content/cloud-servers/resize-up-compute-and-memory-servers.md [hunk 2]
```diff
@@ -70,16 +72,51 @@ Use the following steps to resize up your servers:
...
 - [Basic Cloud Server security](/how-to/basic-cloud-server-security)
-- [Reboot your server](/how-to/reboot-your-server)
+- [Reboot your server](/how-to/reboot-a-server)
```
Comment (RackerKate) [inline content/cloud-servers/resize-up-compute-and-memory-servers.md]: Suggest matching link text to destination page name. It changed a bit.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2be8b61849/how-to/).

**Pull request merged**